src/theme/Footer/index.tsx

```diff
@@ -163,24 +164,29 @@ function Footer(): JSX.Element | null {
               </ul>
             </li>
             <li>
-              <span>Community</span>
+              <span>Ecosystem</span>
               <ul>
                 <li>
-                  <Link href="https://minaprotocol.com/community">Welcome</Link>
+                  <Link href="https://minaprotocol.com/community">Community</Link>
                 </li>
                 <li>
-                  <Link href="https://minaprotocol.com/genesis">
-                    Genesis Program
+                  <Link href="https://minaprotocol.com/grants">
+                    Grants
                   </Link>
                 </li>
                 <li>
-                  <Link href="https://minaprotocol.com/leaderboard">
-                    Testnet Leaderboard
+                  <Link href="https://minaprotocol.com/work-with-mina">
+                    Work with Mina
                   </Link>
                 </li>
                 <li>
-                  <Link href="https://minaprotocol.com/work-with-mina">
-                    Careers
+                  <Link href="https://minafoundation.notion.site/minafoundation/Mina-Community-Blog-91b8fdfd1b9e444ca55ddd1b90bade3c">
```

Comment:
```suggestion
                  <Link href="https://minaprotocol.com/blog">
```

Reply:
let's be cautious about including official links to unofficial blogs 
(the URL you provide is a community member)

Reply:
Ayee, I just uplifted the links directly from Mina's footer and it seemed to be like an MVP version or something but yeah thanks!